docs/MVS_parameters_list.csv

```diff
@@ -23,7 +23,8 @@ None,Latitude coordinate of the project's geographical location.,45.641603,Shoul
 None,Number of operational years of the asset until it has to be replaced.,30,Natural number,numeric,Year,lifetime,lifetime-label,conversion;production;storage_csv;fixcost,
 None,Longitude coordinate of the project's geographical location.,10.95787,Should follow geographical convention,numeric,None,longitude,longitude-label,project_data,
 None,The maximum amount of total emissions in the optimized energy system.,100000,Acceptable values are either a positive real number or None,numeric,kgCO2eq/a,maximum_emissions,maxemissions-label,constraints,
-None,The maximum installable capacity for an asset.,1000,Acceptable values are either a positive real number or None,numeric,kWp,maximumCap,maxcap-label,production,
+None,The maximum additional capacity for an asset,1000,Acceptable values are either a positive real number or None,numeric,kWp,maximumAddCap,maxaddcap-label,production,
```

Comment:
See above, I am not sure if this should be in here, as it is not something that the user can actually define. @Bachibouzouk ?